tests/test_functional/test_aipw.py

```
@@ -42,11 +77,31 @@ def test_compute_aipw_att(self):
+        self.assertAlmostEqual(att_aipw[EFFECT], self.true_att, delta=0.01)
 
 
 class TestAIPW_ATT_ps_misspecified(TestAIPW_ATT_base):
-    alpha = [0.1, 0.2, -0.3, 1]
+    alpha = [
+        0.1,
+        0.2,
+        -0.3,
+        10,
+        5,
+    ]  # evem though the ps is misspecified, the adjustment gives as a correct effect
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)